types/react-data-grid/index.d.ts

```diff
@@ -359,15 +381,15 @@ declare namespace AdazzleReactDataGrid {
     /**
      * Information about a row update
      */
-    interface RowUpdateEvent {
+    interface RowUpdateEvent<T> {
```

Comment:
It looks like adding generic parameters to these Events is a breaking change (since `RowUpdateEvent` is no longer a valid type, `RowUpdateEvent<any>` is). Would you be OK with making generic parameters optional (i.e. `RowUpdateEvent<T = any>`) for backwards compatibility?

I know this'll be a major version bump, but I don't know what major version bumps imply about breaking changes to typings.

Reply:
Yes, adding a default generic parameter looks like the way to go regarding backwards compatibility. I'll just make sure to include the proper/updated use in the comments above.

Apparently, DefinitelyTyped versioning follows the versioning of the libraries it supports, so the major and minor versions should be the same, with bugfixes being independent. That being said, we should probably bump the version to 4.0, to signify both the typing and library major version bump.